src/instructlab/model/backends/vllm.py

```diff
@@ -144,3 +133,31 @@ def run_vllm(
     logger.info(f"vLLM starting up on pid {vllm_process.pid} at {api_base}")
 
     return vllm_process
+
+
+def build_vllm_cmd(
+    host: str, port: int, model_path: pathlib.Path, vllm_args: list[str]
+) -> list[str]:
+    """
+    Build the vLLM command to run the server.
+    """
+    vllm_cmd = [sys.executable, "-m", "vllm.entrypoints.openai.api_server"]
+
+    if "--host" not in vllm_args:
+        vllm_cmd.extend(["--host", host])
```

Comment:
Users can pass arguments `["--host", "localhost"]` and as `["--host=localhost"]`. This trick should solve the problem:

```python
arg_names = set(e.split("=", 1)[0] for e in vllm_args)
if "--host" not in arg_names:
    ...
```

Reply:
That's a good point. Thanks for the snippet!